e2e/kube.go

```diff
@@ -59,8 +51,28 @@ func getClusterKubeClient(ctx context.Context, resourceGroupName, clusterName st
 	restConfig.GroupVersion = &schema.GroupVersion{
 		Version: "v1",
 	}
+	// it's test cluster avoid unnecessary rate limiting
```

Comment:
were you affected by rate limiting during tests ?  I'm quite surprised we are issuing that many requests within a short time span

Reply:
```
DefaultQPS: 5
DefaultBurst: 10
```

Yes, I hit it. I think when we probably run more than 5 tests concurrently on the same cluster. And we poll status with a second interval.

Reply:
I'm not concerned with overloading k8s api